application/core/services/novel_service.py

```
@@ -358,6 +399,8 @@ def update_novel(
                 novel.generation_prefs, generation_prefs
             )
 
+        self.novel_repository.save(novel)
+        self._invalidate_novels_cache()
         # 增量 patch：避免全量 save 把 autopilot_status 等未改字段写回 stopped
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)